src/components/common/Footer.vue

```diff
@@ -169,26 +144,30 @@ export default {
   }
   .footer-content-text{
     font-size:14px;
-    font-family:SourceSansPro-Regular;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Hiragino Sans GB',
+  'Microsoft YaHei', 'Helvetica Neue', Helvetica, Arial, sans-serif, 'Apple Color Emoji',
+  'Segoe UI Emoji', 'Segoe UI Symbol';
     font-weight:400;
     color:#AFACAC !important;
     line-height:18px;
     cursor: pointer;
   }
   .footer-content-text-active{
     font-size:14px;
-    font-family:SourceSansPro-Regular;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Hiragino Sans GB',
+  'Microsoft YaHei', 'Helvetica Neue', Helvetica, Arial, sans-serif, 'Apple Color Emoji',
+  'Segoe UI Emoji', 'Segoe UI Symbol';
     font-weight:400;
     color:#595757 !important;
     line-height:18px;
     cursor: pointer;
   }
   .footer-content-text:hover{
-    color:#32A4BE !important;;
+    color:#002766 !important;;
     text-decoration:underline !important;
   }
   .footer-content-text:active{
-    color:rgba(50,164,190,1) !important;;
+    color: #002766 !important;;
     text-decoration:underline !important;
   }
   .footer-language{
```

Comment:
Seems lines 173-180 could be removed. We have removed .footer-language and .footer-language-content

Reply:
removed